azure-pipelines.yml

```
@@ -34,14 +34,29 @@ pr:
 stages:
            
   - stage: system_tests_win64
+    dependsOn: []
     pool:
       name: nimi-bot
       demands:
         - nimi-bot -equals True
-        - agent.os -equals Windows_NT
+        - platform -equals win64 # Run system tests using python 64 bit installed on windows 64 bit
```

Mind spelling - this is a public facing repository.

```suggestion
        - platform -equals win64 # Run system tests using 64-bit Python interpreter on a 64-bit Windows 10 system.
```